apps/publisher/src/app.tsx

```diff
@@ -63,6 +63,10 @@ const MainSourceId = 'Main';
 const DisplayShareSourceId = 'DisplayShare';
 type PublisherState = SourceState;
 
+const date = new Date();
+const streamName =
```

Comment:
@dubeyShivank  do you know why it was working in V1 release but not here?

Reply:
@dubeyShivank  I found the root cause, the date will be recreated when refresh, so we only need to move the date out of the `App` component, then it should work. Don't worry, I will fix it in this PR.